### PR TITLE
Deprecate Python 3.7.x (EOL)

### LINKED ISSUE #784: Dependency Deprecation Alert
Please make sure the following dependencies are removed from buildpack on time.

- Name: python
Version Line: 3.7.x
Date: 2023-06-27
Link: https://www.python.org/dev/peps/pep-0537/

### PR DESCRIPTION
**Pull request by brayanhenao**
Fix #784

### CODE REVIEW

**Comment by brayanhenao** [comment]
Don't merge until #790 is merged (fix for tests)

**Review by robdimsdale — APPROVED** [review]

**Pull request merged**
